extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/CommonPanacheQueryImpl.java

```diff
@@ -26,6 +29,17 @@
     private static final Pattern SELECT_PATTERN = Pattern.compile("^\\s*SELECT\\s+((?:DISTINCT\\s+)?[^\\s]+)\\s+([^\\s]+.*)$",
             Pattern.CASE_INSENSITIVE);
 
+    private interface NonThrowingCloseable extends AutoCloseable {
+        @Override
+        void close();
```

Comment:
Can we make this a `default`? Would some typing later on :)

Reply:
It's only used here, I don't see the point. It's in the JDK that this interface belongs :(

Reply:
True